reference/src/glossary.md

```diff
@@ -1,5 +1,17 @@
 ## Glossary
 
+### ABI (of a type)
+[abi]: #abi-of-a-type
+
+The *function call ABI* or short *ABI* of a type defines how it is passed *by-value* across a function boundary.
```

Comment:
Maybe call this "calling convention" instead? The ABI is the combination of the memory layout, calling convention, linkage model and every other part of the interface between compiled object files, dynamic libraries or the kernel.

Reply:
"ABI" is the term used in existing documents such as [this one](https://rust-lang.github.io/rfcs/3391-result_ffi_guarantees.html). I wouldn't mind a different term but that might be fighting windmills...

Reply:
Calling convention isn't quite correct either, because two types can have the same calling convention but not the same layout, and what's meant when discussing a type's function call ABI is the exact set of properties needed to ensure two function signatures are ABI compatible, which includes both calling convention and layout.